src/Resources/contao/dca/tl_survey.php

```diff
@@ -93,5 +93,16 @@
                 // Keep this for MySQL Strict mode. Otherwise Contao would save an empty string
                 'sql' => ['type' => 'boolean', 'default' => false],
             ],
+            'store_records' => [
+                'inputType' => 'checkbox',
+                'filter' => true,
+                'eval' => [
+                    'tl_class' => 'w50 m12',
```

Comment:
Should this get a clear as well? So that it always stays in the line even if `frozen` is removed from the palette?
```suggestion
                    'tl_class' => 'clr w50 m12',
```